profiles/source/dictionary/schema.yaml

```diff
@@ -263,6 +281,33 @@ tableSchemaMissingValues:
       {
         "missingValues": []
       }
+tableSchemaExamples:
+  title: Examples
+  description: Links to example data files
+  type: array
+  minItems: 0
+  items:
+    "$ref": "#/definitions/tableSchemaExample"
+  examples:
+    - |
+      {
+        "examples": [
+          {
+            "title": "Valid data",
+            "path": "http://example.com/valid-data.csv"
+          }
+        ]
+      }
+tableSchemaExample:
+  title: Example
+  description: Link to an example data file
+  type: object
+  properties:
+    title:
+      "$ref": "#/definitions/title"
```

Comment:
```suggestion
      "$ref": "#/definitions/name"
```

Reply:
Would you mind keeping both `title` and `name` ? Both are present in data resources (`name` is mandatory), but as mentioned in another comment, we use `title` to give hints about what is interesting about the given example. This is much less easy with only the `title`